src/plugin-host/plugin-events/plugin-event-types.ts

```diff
@@ -0,0 +1,9 @@
+export enum PLUGIN_EVENT
```

Comment:
I don't see the `NoArgumentsPluginEventArguments` here. Does that mean it is not used at all, then?

Reply:
`NoArgumentsPluginEventArguments` is a `PluginEventArguments`, not a `PluginEvent`.